src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -346,7 +356,7 @@ int CreateRunRequest(int argc, char** argv, const char** env,
     req->add_command(NormalizedRelativePath(cwd, output));
   }
   set<string> inputs;
-  int compute_input_res = ComputeInputs(argc, argv, env, cwd, cmd_id, is_compile, &inputs);
+  int compute_input_res = ComputeInputs(argc, argv, env, cwd, cmd_id, is_compile, is_javac, &inputs);
```

Comment:
Could we use this to force run remote instead of using RUN_ALL_REMOTELY?

Reply:
Good point, done.